main.css

```diff
@@ -85,6 +97,7 @@ body {
 
   .prog-table {
```

Comment:
Precedent @roryc89 and I set on Friday:

BEM component | Indentation
------ | ------
Block | 0 (zero)
Element | 2 
Modifier (on a Block) | 4
Modifier (on an Element) | 6

e.g.:
```css
.block {}
    .block--modifier {}
  .block__element1 {}
  .block__element2 {}
      .block__element2--modifier1 {}
      .block__element2--modifier1 {}
  .block__element3 {}
.another-block